tools/pipelines/test-perf-benchmarks.yml

```
@@ -197,6 +198,19 @@ stages:
               ls -laR ${{ variables.consolidatedTestsOutputFolder }};
               node --require @ff-internal/aria-logger bin/run --handlerModule $(toolAbsolutePath)/dist/handlers/executionTimeTestHandler.js --dir '${{ variables.consolidatedTestsOutputFolder }}';
 
+        - task: Bash@3
+          displayName: Send Execution Time Performance Benchmark Measurments to Azure App Insights
+          inputs:
+            targetType: 'inline'
+            workingDirectory: $(toolAbsolutePath)
+            script: |
+              echo "Writing performance benchmark output to Azure App Insights..."
+              ls -laR ${{ variables.consolidatedTestsOutputFolder }};
+              node --require @ff-internal/aria-logger bin/run appInsights --handlerModule $(toolAbsolutePath)/dist/handlers/appInsightsExecutionTimeTestHandler.js --dir '${{ variables.consolidatedTestsOutputFolder }}' --connectionString '$(fluid-app-insights-connection-string)';
```

```suggestion
              node bin/run appInsights --handlerModule $(toolAbsolutePath)/dist/handlers/appInsightsExecutionTimeTestHandler.js --dir '${{ variables.consolidatedTestsOutputFolder }}' --connectionString '$(fluid-app-insights-connection-string)';
```
The new steps don't need to write anything to Aria/Kusto so we don't need that `--require`. Let's remove it from the other new steps too.

---

Done in recent commit # 84045